schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -62,30 +67,33 @@
             "pattern": "^CVE-[0-9]{4}-[0-9]{4,19}$"
         },
         "orgId": {
-            "type": "string",
-            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+            "description": "a UUID for an organization participating in the CVE program. This UUID can be used to lookup the organization record in the user registry service.",
+            "$ref": "#/definitions/uuidType"
         },
         "userId": {
-            "type": "string",
-            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+            "description": "a UUID for a user participating in the CVE program. This UUID can be used to lookup the user record in the user registry service.",
+            "$ref": "#/definitions/uuidType"
         },
         "shortName": {
```

Comment:
Should we remove shortName, since this can be looked up in the user registry by the uuid?